resources/syntax/Jinja.sublime-syntax

```diff
@@ -37,6 +37,33 @@ variables:
         |endfilter
       )\b
     )
+  builtin_filters: |-
+    (?x:\b(?:
```

Comment:
IMHO, I think you can have the opening word boundary match `\b(?:` on the next line just to maintain consistency with other syntax variables. Similarly for `builtin_tests`

Reply:
Yep, true. I think my reasoning had something to do with line length.